src/components/Input/DropDown/DropDownInput.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+import colors from "../../../theme/colors";
+
+const StyledDropDownItem = styled.span`
+  .dropdown__items & {
+    appearance: none;
+    height: 36px;
+    margin-left: 8px;
+    margin-right: 8px;
+    margin-top: 4px;
+    padding-left: 12px;
+    padding-right: 12px;
+    font-size: 16px;
+    text-align: left;
+    border: none;
+    background-color: ${colors.white.base};
+    display: flex;
+    align-items: center;
+    &:hover {
+      background-color: ${colors.azure.base};
+      border-radius: 2px;
+      color: white;
+    }
+    &:focus {
+      background-color: ${colors.azure.base};
+      border-radius: 2px;
+      color: white;
+      outline: none;
+    }
+    &.dropdown__selected {
+      background-color: ${colors.onyx.muted};
+      border-radius: 2px;
+      color: white;
```

Comment:
Many of the styles within these pseudo-selectors seem redundant. You can define the common styles together and the unique styles separately to simplify maintenance, e.g.:

```
    &:hover, &:focus, &.dropdown__selected {
      border-radius: 2px;
      color: white;
    }

    &:hover, &:focus {
      background-color: ${colors.azure.base};
    }

    &.dropdown__selected {
      background-color: ${colors.onyx.muted};
    }

    &:focus {
      outline: none;
    }
```